.github/workflows/ci-cd.yml

```
@@ -83,37 +83,53 @@
         run: |
           echo "🧪 Executando todos os testes..."
 
-          # Executar testes unitários e de arquitetura
+          # Executar testes unitários e de arquitetura (EXCLUINDO código gerado do compilador)
           dotnet test tests/MeAjudaAi.Shared.Tests/MeAjudaAi.Shared.Tests.csproj \
             --configuration Release --no-build --verbosity normal \
-            --collect:"XPlat Code Coverage" --results-directory TestResults/Shared
+            --collect:"XPlat Code Coverage" --results-directory TestResults/Shared \
+            -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.ExcludeByFile=\
+"**/*OpenApi*.generated.cs,**/System.Runtime.CompilerServices*.cs,**/*RegexGenerator.g.cs"
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)